main.m

```diff
@@ -13,138 +13,64 @@
 [pathRepoFolder,~,~] = fileparts(pathRepo);
 
 %% Initialize S
-pathDefaultSettings = fullfile(pathRepo,'DefaultSettings');
-addpath(pathDefaultSettings)
+addpath(fullfile(pathRepo,'DefaultSettings'))
 
 [S] = initializeSettings('Falisse_et_al_2022');
```

Comment:
Why not do it like this?
```suggestion
% name of the subject
S.subject.name = 'Falisse_et_al_2022';
[S] = initializeSettings(S.subject.name);
```

Reply:
`[S] = initializeSettings('Falisse_et_al_2022');` can also be useful for other subjects, if they use a comparable model.

Reply:
Would it not be best practice to always have a settings file for each subject?

Reply:
I think that this may depend on your study / application.
I usually initialise settings with 'Falisse_et_al_2022' and I only add in the settings file for each subject those that are different from default or from 'Falisse_et_al_2022'.
I would keep it as it is now